.github/workflows/go.yml

```diff
@@ -59,8 +105,148 @@ jobs:
+        mkdir build/
+        go build -o build/mackerel-agent.exe -ldflags="-X main.gitcommit=$COMMIT" github.com/mackerelio/mackerel-agent
+        go build -o build/mackerel-agent-kcps.exe -ldflags="-X main.gitcommit=$COMMIT -X github.com/mackerelio/mackerel-agent/config.apibase=http://198.18.0.16" github.com/mackerelio/mackerel-agent
+      env:
+        GOARCH: ${{ matrix.GOARCH }}
+        CC: ${{ matrix.CC }}
+
+    - name: Build mackerel-agent-plugins
+      run: |
+        foreach ($line in (Get-Content wix/pluginlist.txt) -as [string[]]) {
+          $piece = $line -split "/"
+          $outputFilename = $piece[$piece.Length - 1]
+          Write-Host $outputFilename
+          go build -o build/$outputFilename.exe $line
+        }
+      env:
+        GOARCH: ${{ matrix.GOARCH }}
+        CC: ${{ matrix.CC }}
+
+    - name: Build tools
+      run: |
+        go build -o build/wrapper.exe wix/wrapper/wrapper_windows.go wix/wrapper/install.go
+        go build -o build/replace.exe wix/replace/replace_windows.go wix/replace/shell_windows.go
+        go build -o build/generate_wxs.exe wix/generate_wxs/generate_wxs.go
+      env:
+        GOARCH: ${{ matrix.GOARCH }}
+        CC: ${{ matrix.CC }}
+
+    - name: Upload build artifacts
+      uses: actions/upload-artifact@v2
+      with:
+        name: windows-build-artifacts-${{ matrix.PLATFORM_ID }}
+        path: |
+          build/
+
+  windows-installer:
+    name: Make Installer (Windows)
+    runs-on: windows-2019
+    needs: build-windows
+    if: github.ref == 'refs/heads/master'
+    strategy:
+      matrix:
+        include:
+          - PLATFORM_ID: x86
+          - PLATFORM_ID: x64
+            MSI_SUFFIX: -x64
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/download-artifact@v2
+      with:
+        name: windows-build-artifacts-${{ matrix.PLATFORM_ID }}
+        path: build/
+
+    - name: Get gobump binary
+      run: |
+        $ProgressPreference = 'SilentlyContinue'
+        Invoke-WebRequest -Uri https://github.com/x-motemen/gobump/releases/download/v0.2.0/gobump_v0.2.0_windows_amd64.zip -OutFile ~/gobump.zip
+        Expand-Archive -Path ~/gobump.zip -DestinationPath ~/
+
+    - name: Build Installer
+      run: |
+        $INFO = (~/gobump_v0.2.0_windows_amd64/gobump.exe show | ConvertFrom-Json)
+        Write-Host $INFO
+        $VERSION = $INFO.version
+        Write-Host $VERSION
+        if ($null -eq $VERSION) {
+          exit 1
+        }
+        & "./wix/build.bat" "$VERSION"
+      env:
+        PLATFORM_ID: ${{ matrix.PLATFORM_ID }}
+        MSI_SUFFIX: ${{ matrix.MSI_SUFFIX }}
+
+    - name: Signing Installer
+      run: |
+        mkdir $env:RUNNER_TEMP/sign/
+        $bytes = [Convert]::FromBase64String("${{ secrets.WINDOWS_CERT_P12_BASE64 }}")
+        [IO.File]::WriteAllBytes("$env:RUNNER_TEMP/sign/cert.p12", $bytes)
+        $SIGNTOOL = "C:/Program Files (x86)/Windows Kits/10/bin/10.0.19041.0/x86/signtool.exe";
```

Comment:
Hmm, it looks like this path will change when Windows SDK will be updated... https://developer.microsoft.com/ja-jp/windows/downloads/sdk-archive/
Is there any stable path for this tool?

Reply:
Every time the Windows SDK is added, it seems to be installed in the environment, and it seems to be fixed in this version. 

At the moment, there are 6 versions of the WIndows10 SDK installed in windows-2019 environment.
https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md#workloads-components-and-extensions